tools/packaging/packaging.strings.psd1

```diff
@@ -121,7 +121,7 @@ open {0}
         <authors>Microsoft</authors>
         <owners>Microsoft,PowerShell</owners>
         <requireLicenseAcceptance>true</requireLicenseAcceptance>
-        <description>PowerShell runtime for hosting PowerShell Core</description>
+        <description>Runtime for hosting PowerShell</description>
```

Comment:
I'm not sure that this is understandable.

Reply:
cc @joeyaiello, seems clear to me